deps/build.jl

```diff
@@ -7,6 +7,13 @@ builddir = joinpath(BinDeps.depsdir(mpi), "build")
 prefix = joinpath(BinDeps.depsdir(mpi), "usr")
 src = joinpath(BinDeps.depsdir(mpi), "src")
 
+if isdir(builddir)
```

Comment:
We could use `ispath` instead of `isdir` but I guess it is quite rare that `build` and `usr` would be anything but directories here.